### ISSUE TITLE
wallet stopped at block 7227

### ISSUE THREAD

**Issue by x23101983** [FortuneBlockTeam/fortuneblock#2]
wallet stopped at block 7227.
the last update i did was 3 days ago . v1.0.1 Latest
@accarto accarto released this 3 days ago
v1.0.1
d5db797.
Is there a new update?

**Comment by accarto** [comment]
We are working on a permanent solution. In the meantime, you can delete all directories except the 'wallets' directory, then restart the node with the following parameters:  
`./fortuneblockd -addnode=15.204.9.173:27777 -addnode=158.179.16.228 -daemon`.

**Comment by x23101983** [comment]
Ok thanks now it is updated with these settings. But unfortunately all the blocks I found were not confirmed so I wasted a lot of electricity for nothing and the electricity here in Italy is the most expensive in the world unfortunately. It is already very difficult to compete with Rplant and these interruptions make it even more difficult. I hope you solve this problem once and for all. 3 days ago I mined for 24 hours and all the blocks were not confirmed.
If you send me some FTB I would be very grateful, Getting to 60K FTB to make the SmartNode is very difficult.
FbFdSsimVDeB6A26pzHUoZP1vBm5HXCpnL
